dev/io.openliberty.mcp.internal/src/io/openliberty/mcp/internal/schemas/TypeUtility.java

```diff
@@ -338,4 +340,32 @@ public static boolean hasGenericParams(Type type) {
         return false;
     }
 
+    public static Type createCustomType(Type type, Map<TypeVariable<?>, Type> genericMap) {
+        if (type instanceof TypeVariable<?> tv) {
+            if (genericMap.get(tv) == null) {
+                return type;
+            } else {
+                return genericMap.get(tv);
+            }
```

Comment:
I don't recall what can go in the generic map, do we need to call it in a loop or not?